README.md

```diff
@@ -33,14 +37,16 @@ Alternatively, you can install ``PAINT`` locally. To achieve this, there are two
 The ``PAINT`` repository is structured as shown below:
 ```
 .
-├── paint # Parent package
-│   ├── data # Objects in the field, e.g. heliostats and receivers
-│   └── util
-├── plots # scripts to create summary plots of the data
-├── tests/
+├── data-preparation-scripts #scripts used to generate STAC files and structure data
+├── html #code for the paint-database.org website
+├── markers #saved markers for the WRI1030197 power plant in Jülich
+├── paint #python package/
 │   ├── data
 │   └── util
-└── tutorials # small use example for the PAINT data API
+├── plots #scripts to generate plots
```

Comment:
Is there a reason why you removed the spaces before the comments? I think it would look a bit cleaner with them :).

Reply:
Nope, just forgot about them. I've added the spaces back in